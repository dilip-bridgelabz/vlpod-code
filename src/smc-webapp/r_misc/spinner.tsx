/*
 *  This file is part of CoCalc: Copyright © 2020 Sagemath, Inc.
 *  License: AGPLv3 s.t. "Commons Clause" – see LICENSE.md for details
 */

import * as React from "react";
import { Icon } from "./icon";

export function Spinner() {
  return <Icon name="spinner" spin={true} />;
}
