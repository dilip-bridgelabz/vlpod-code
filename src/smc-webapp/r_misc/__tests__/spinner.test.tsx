/*
 *  This file is part of CoCalc: Copyright © 2020 Sagemath, Inc.
 *  License: AGPLv3 s.t. "Commons Clause" – see LICENSE.md for details
 */

import * as React from "react";
import { shallow } from "enzyme";
import { Spinner } from "../spinner";

test("smoke test", () => {
  const rendered = shallow(<Spinner />);
  expect(rendered).toMatchSnapshot();
});
