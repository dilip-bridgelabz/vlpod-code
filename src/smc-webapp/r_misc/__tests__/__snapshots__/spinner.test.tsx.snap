// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke test 1`] = `
<Icon
  name="spinner"
  spin={true}
/>
`;
